config/config.go

```diff
@@ -402,6 +410,10 @@ var defaultConf = Config{
 		MaxBatchWaitTime:  0,
 		BatchWaitSize:     8,
 	},
+	TiFlash: TiFlash{
```

Comment:
Can we get region info about TiFlash store from PD when user configs it to other value?